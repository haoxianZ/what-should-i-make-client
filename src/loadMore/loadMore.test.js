import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LoadMore from './loadMore';

describe(`Load More component`, () => {
  it('renders without crash', () => {
    const wrapper = shallow(<LoadMore />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
}) 