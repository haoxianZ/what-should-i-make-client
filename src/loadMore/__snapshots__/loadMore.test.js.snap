// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Load More component renders without crash 1`] = `
<section
  className="loadMore"
/>
`;
